stdpopsim/models.py

```diff
@@ -192,3 +200,15 @@ def all_models():
         if mod.startswith("stdpopsim"):
             ret.append(cls())
     return ret
+
+
+def all_model_generation_times():
```

Comment:
What's the idea of this function? It's simple to reproduce (``[model.generation_time for model in stdpopsim.all_models()]``), so I don't see the need to have it.

Reply:
this was just a helper function I added in my struggles with codecov. it's definitely bloat. 

related-- is there a simple way to run codecov locally?

Reply:
You can run coverage locally using some arguments to nose like [here](https://github.com/popgensims/stdpopsim/blob/master/.circleci/config.yml#L32). This will tell you how many lines you're missing and stuff, but won't give the detailed line-by-line analysis.